sources/packages/backend/libs/integrations/src/esdc-integration/e-cert-integration/e-cert-part-time-integration/_tests_/unit/e-cert-part-time-integration.service.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { createMock } from "@golevelup/ts-jest";
+import { SshService } from "@sims/integrations/services";
+import { DisbursementValueType, RelationshipStatus } from "@sims/sims-db";
+import { ConfigService } from "@sims/utilities/config";
+import { ECertPartTimeFileFooter } from "../../e-cert-files/e-cert-file-footer";
+import { ECertPartTimeFileHeader } from "../../e-cert-files/e-cert-file-header";
+import { ECertPartTimeIntegrationService } from "../../e-cert-part-time.integration.service";
+
+describe("ECertPartTimeIntegrationService-createRequestContent", () => {
+  let eCertPartTimeIntegrationService: ECertPartTimeIntegrationService;
+  let eCertPartTimeFileHeader: ECertPartTimeFileHeader;
+  let eCertPartTimeFileFooter: ECertPartTimeFileFooter;
```

Comment:
It seems that these objects are not needed at this level and they can be declared inside the `beforeAll`.